src/devices/Pn532/P3.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Pn532
+{
+    /// <summary>
+    /// The P3 statte
+    /// </summary>
+    [Flags]
+    public enum P3
```

Comment:
please expand this name or make it internal since this is far from obvious what this means

Reply:
Those are exposed for most of them IO that can be used. For example to trigger another sensor. Will add some more explanations.

Reply:
I see, perhaps just rename to `Port3` and similarly for others